### PR TITLE
Folder integration

### LINKED ISSUE #4: Task Folder UI
- Have a dummy subject name for the folder name
- Make a fire image next to the name with a counter for the streaks
- Description box
- Line separating the sections
- Section for the today/upcoming tasks

### LINKED ISSUE #36: Gamification of tasks (maybe some streaks feature like Snapchat)

### PR DESCRIPTION
**Pull request by jlowe11**
TODO: Put in folder_id for Route Navigation

### CODE REVIEW

**Comment by jlowe11** [comment]
Pulled from develop

File: packages/mongoose-database/index.js
Comment (ishaansathaye) [inline packages/mongoose-database/index.js:1]: probably should remove any changes from index.js so no conflicts

**Review by ishaansathaye — APPROVED** [review]
lgtm.

**Review by kct417 — COMMENTED** [review]
this package.json should not have any dependencies. all of them should be in the respective folders. ie, dotenv should be in backend, and mongoose should be in database folders. not sure why we have env as well when we are already using dotenv

**Pull request closed without merging**